### ISSUE TITLE
Latex?

### ISSUE THREAD

**Issue by rlvesco7** [kelvinh/org-page#97]
Is it possible to use mathjax with org-page? It doesn't seem to process inline latex $$\frac{4}{5}$$. 

If I export a page using org-html export function it works fine. 

This would be a very useful feature if it doesn't already exist. 

Thanks for the great work.

**Comment by kuangdash** [comment]
Open the file "/path/to/org-page-page/theme/header.mustache"
add the following codes before </head> label:
```
<script type="text/javascript"
          src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
 </script>
```
May it be done……

**Comment by kelvinh** [comment]
@kuangdash Thanks for your reminder, now implemented in `mdo` theme.

@rlvesco7 You can use `mdo` theme for Tex/LaTeX rendering now, but if you want some customization, you may need to create your own theme from `mdo`, and customize variables according to MathJax doc: http://docs.mathjax.org/en/latest/options/tex2jax.html#configure-tex2jax
